drivers/gpu/drm/rcar-du/rcar_du_crtc.c

```diff
@@ -858,6 +859,41 @@ int rcar_du_crtc_create(struct rcar_du_group *rgrp, unsigned int index)
 		setup_gfx_clk = true;
 	}
 
```

Comment:
We won't never be here if we jump to gfx_skip. You have to put gfx_skip label here.

Reply:
My bad, I misunderstood usage of that label.